src/client-functions/selectors/add-api.js

```diff
@@ -10,34 +9,33 @@ import makeRegExp from '../../utils/make-reg-exp';
 import selectorTextFilter from './selector-text-filter';
 import selectorAttributeFilter from './selector-attribute-filter';
 
-const SNAPSHOT_PROPERTIES = NODE_SNAPSHOT_PROPERTIES.concat(ELEMENT_SNAPSHOT_PROPERTIES);
+let apiFn = null;
```

Comment:
Can we avoid using the global variable for storing temporary results? You can create an internal class (e.g. `SelectorAPIBuilder`) if you have to share context between functions.